builders/integrations/relayers/gelato.md

```diff
@@ -0,0 +1,235 @@
+
+```
+  const test_token = "0xEeeeeEeeeEeEeeEeEeEeeEEEeeeeEeeeeeeeEEeE";
+  // Create Mock wallet
+  const wallet = Wallet.createRandom();
+  const sponsor = await wallet.getAddress();
+  console.log(`Mock PK: ${await wallet._signingKey().privateKey}`);
+  console.log(`Mock wallet address: ${sponsor}`);
+```
+
+
+#### Add Request Data {: #add-request-data }
+
+In this step we have to provide the ABI-encoded call data for the function we want to interact with. You can generate this by taking the following steps:
+
+1. Navigate to the **Write Contract** heading of the [Hello World Contract on Moonscan](https://moonscan.io/address/0x3456E168d2D7271847808463D6D383D079Bd5Eaa#writeContract){target=_blank} 
+2. Press  **Connect to Web3**. After you accept the terms and conditions, you can connect your wallet
+3. Head to the `sayHiVanilla` function and provide the following default value for the `_feeToken` parameter: `0xEeeeeEeeeEeEeeEeEeEeeEEEeeeeEeeeeeeeEEeE`
+4. Press **Write**
+5. Without confirming the transaction in MetaMask, click on the **Hex** tab
+6. Press **Copy Raw Transaction Data**
+
+The resulting ABI-encodeded call data should look like `0x4b327067000000000000000000000000eeeeeeeeeeeeeeeeeeeeeeeeaeeeeeeeeeeeeeeeee`
+
+![Gelato Relay SDK](/images/builders/integrations/relayers/gelato/gelato-5.png)
+
+There are some additional parameters defined here, such as paymentType, maxFee, and gas limit. There are a variety of possible [payment types](https://docs.gelato.network/developer-products/gelato-relay-sdk/payment-types){target=_blank} you can choose from. For simplicity, replay protection has not been considered in this example. 
+
+```
+  // abi encode for HelloWorld.sayHiVanilla(address _feeToken)
+  const data = `0x4b327067000000000000000000000000eeeeeeeeeeeeeeeeeeeeeeeeaeeeeeeeeeeeeeeeee`;
+  
+  // Async Gas Tank payment model (won't be enforced on testnets, 
+  // hence no need to deposit into Gelato's Gas Tank smart contract)
+  const paymentType = 1;
+  // Maximum fee that sponsor is willing to pay worth of test_token
+  const maxFee = "1000000000000000000";
+  // Gas limit
+  const gas = "200000";
+  
+  // We do not enforce smart contract nonces to simplify the example.
+  // In reality, this decision depends whether or not target
+  // address already implements replay protection. (More info in the docs)
+  const sponsorNonce = 0;
+  const enforceSponsorNonce = false;
+  // Only relevant when enforceSponsorNonce=true
+  const enforceSponsorNonceOrdering = false;
+```
+
+#### Putting it All Together {: #putting-it-all-together }
+
+The last few steps are building the request object, hashing it, and finally, signing it. The last step is to submit the request and the signature to the Gelato Relay API. You can copy and paste the below code into a javascript file. You can name the file `hello-world.js` or a similar name. 
```

Comment:
Since there is code being assembled in the previous section, I think creating the file should happen first and then added to the file as you go... so maybe the `forwardRequestExample` function should be created in previous section then added to in this one